deps/verifier/src/snp/mod.rs

```diff
@@ -84,8 +85,10 @@ impl Verifier for Snp {
             cert_chain,
         } = serde_json::from_slice(evidence).context("Deserialize Quote failed.")?;
 
-        let Some(cert_chain) = cert_chain else {
-            bail!("Cert chain is unset");
+        let cert_chain = if let Some(chain) = cert_chain{
```

Comment:
So the cert chain is type `Option<Vec<Cert>>` or something. This accounts for cases where the option is none, but what if the option is there but the vector is empty?

Ideally we should get the vcek even if there is a non-empty cert chain as long as the vcek/vlek isn't part of it.

Reply:
Could this be part of the future follow-up PR?

Reply:
We can leave check the cert types to a follow-up, but please add a case to this statement to catch an empty vector.

Reply:
Added the empty vector check